src/challenges/2.1.md

```diff
@@ -91,7 +91,7 @@ screen.getByText(/world/i) // substring match, ignoring case
 <details>
   <summary>💡 Get Hint</summary>
   
-  When a sentence is split into different nodes, Testing Library can't find the sentece. Try using regex to look for a part, and then access the `parentElement` to check its `textContent`.
+  When a sentence is split into different nodes, the Testing Library can't find the sentence. Try using regex to look for a part, and then access the `parentElement` to check its `textContent`.
```

Comment:
"Testing Library" é um nome, tipo "Joao". nesse casos nao precisa do "the" atrás, right?